scripts/main.js

```diff
@@ -403,6 +403,9 @@ $(document).ready(async function(){
       }
     `);
 
+    // ensure the list of regions is alphabetized
+    states.regions.sort((a, b) => {a.name.localeCompare(b.name)})
```

Comment:
This works, but I wonder if we can utilize a `SortByInput` https://docs.mongodb.com/realm/graphql/types-and-resolvers/#sortbyinput in the graphql query to have the server do this work for us.  If we have multiple clients in the future (or even different calls to "get a list of ordered states" in the same app), they can utilize the same graphql query and not have to reimplement the sorting logic themselves

Reply:
ooo that's a good point - I like the `sortBy` implementation better, just refactored and re-pushed!